vision/google/cloud/vision/client.py

```diff
@@ -123,3 +61,14 @@ def image(self, content=None, filename=None, source_uri=None):
         """
         return Image(client=self, content=content, filename=filename,
                      source_uri=source_uri)
+
+    @property
+    def vision_api(self):
```

Comment:
I realize we do it elsewhere, but it's somewhat bad form to make this public. Name the `@property` as `_vision_api` and then change the internal cached value to something else, e.g. `_vision_api_interal`.

For example: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/6b93d4066fd85a1f9b56fd10baa5aec6e844f9e5/bigtable/google/cloud/bigtable/client.py#L273